codestodians/food-basket-front-end
# [Feature: #163726651] Fix modal console error

#### What does this PR do?
- Fixes the error shown on modal display for authentication

#### Description of Task to be completed?
- Deprecate the use of static `propTypes` causing the error.
- Make `Header` and `LoginForm` functional components.

#### How should this be manually tested?
- Check out into the `bug/fix-location-error-on-modal-render-163726651` branch.
- Run the app. 
- Click on the Login icon.
- The console should not have errors.

#### Any background context you want to provide?
- It would seem that with the react upgrade, the static `propTypes` idea becomes a problem.

#### What are the relevant pivotal tracker stories?
- [#163726651](https://www.pivotaltracker.com/story/show/163726651)
